plugins/org.jboss.tools.openshift.client/META-INF/MANIFEST.MF

```diff
@@ -23,4 +23,13 @@ Export-Package: com.openshift.client,
  com.openshift.internal.client.httpclient.request,
  com.openshift.internal.client.response,
  com.openshift.internal.client.ssh,
- com.openshift.internal.client.utils
+ com.openshift.internal.client.utils,
+ com.openshift3.client,
+ com.openshift3.client.model,
+ com.openshift3.internal.client
+Import-Package: org.eclipse.debug.internal.core,
+ org.eclipse.osgi.service.debug;version="1.2.0",
+ org.eclipse.pde.internal.core.bundle,
```

Comment:
after verifying: the debug bundle is needed by the Trace while the pde is not required.